.editorconfig

```diff
@@ -5,7 +5,7 @@
 root = true
 
 [*]
-end_of_line = lf
+end_of_line = auto
```

Comment:
please, keep it in `lf`, and use editorconfig plugin in your editor. This is important for preventing people from not viewing  line endings.